### PR TITLE
[Connector-V2] [Chore] Canonical name for AmazonDynamodb

### PR DESCRIPTION
**Pull request by Hisoka-X**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)

### CODE REVIEW

**Review by ashulin — DISMISSED** [review]
LGTM

**Review by TaoZex — DISMISSED** [review]

**Comment by Hisoka-X** [comment]
@ashulin @TaoZex PTAL again, Thanks!

File: docs/en/connector-v2/sink/AmazonDynamoDB.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+| access_key_id     | string | yes      | -             |
+| secret_access_key | string | yes      | -             |
+| table             | string | yes      | -             |
+| batch_size        | string | no       | 25            |
+| batch_interval_ms | string | no       | 1000          |
+| common-options    |        | no       | -             |
+
+### url [string]
+
+url to write to Amazon DynamoDB.
```
Comment (TaoZex) [inline docs/en/connector-v2/sink/AmazonDynamoDB.md]: Other lines is to capitalize the first letter, use the URL is better.

File: docs/en/connector-v2/source/AmazonDynamoDB.md [hunk 2]
```diff
@@ -17,35 +17,35 @@ Read data from Amazondynamodb.
...
-| table    	       | string | yes      | -             |
-| schema           | object | yes      | -             |
-| common-options   |        | yes      | -             |
+| name              | type   | required | default value |
+|-------------------|--------|----------|---------------|
+| url               | string | yes      | -             |
+| region            | string | yes      | -             |
+| access_key_id     | string | yes      | -             |
+| secret_access_key | string | yes      | -             |
+| table    	        | string | yes      | -             |
```
Comment (TaoZex) [inline docs/en/connector-v2/source/AmazonDynamoDB.md]: fix format?

File: docs/en/connector-v2/source/AmazonDynamoDB.md [hunk 3]
```diff
@@ -17,35 +17,35 @@ Read data from Amazondynamodb.
...
+| schema            | object | yes      | -             |
+| common-options    |        | yes      | -             |
 
 ### url [string]
 
-url to read to Amazondynamodb.
+url to read to Amazon Dynamodb.
```
Comment (TaoZex) [inline docs/en/connector-v2/source/AmazonDynamoDB.md]: Same as above.

Comment (Hisoka-X) [reply docs/en/connector-v2/source/AmazonDynamoDB.md]: Done of All

File: docs/en/connector-v2/source/AmazonDynamoDB.md [hunk 4]
```diff
@@ -17,41 +17,41 @@ Read data from Amazondynamodb.
...
-| schema           | object | yes      | -             |
-| common-options   |        | yes      | -             |
+| name              | type   | required | default value |
+|-------------------|--------|----------|---------------|
+| url               | string | yes      | -             |
+| region            | string | yes      | -             |
+| access_key_id     | string | yes      | -             |
+| secret_access_key | string | yes      | -             |
+| table             | string | yes      | -             |
+| schema            | object | yes      | -             |
```
Comment (CalvinKirs) [inline docs/en/connector-v2/source/AmazonDynamoDB.md]: `schema` type is best modified to `config`

Comment (Hisoka-X) [reply docs/en/connector-v2/source/AmazonDynamoDB.md]: Done

**Comment by Hisoka-X** [comment]
@CalvinKirs @TaoZex @ashulin Please review again, Thanks!

**Review by TaoZex — APPROVED** [review]
LGTM

**Review by CalvinKirs — APPROVED** [review]

**Review by EricJoy2048 — APPROVED** [review]

**Pull request merged**